### PR TITLE
draft

### PR DESCRIPTION
**Pull request by jeezOC**

### CODE REVIEW

File: src/context/reducers/authReducer.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { TUserInformation, TAction } from "../types/authTypes";
+
+export const initialUserState = {
+	uid: "",
```
Comment (Chinaski1979) [inline src/context/reducers/authReducer.ts:4]: Fix the indentation of this file. Also in modern JS you are supposed to use single quotes for this specific case.

File: src/context/types/authTypes.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export type TUserInformation = {
+	uid: string;
```
Comment (Chinaski1979) [inline src/context/types/authTypes.ts:2]: fix the indentation of this file

File: src/components/QRReader.tsx [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
   setQrData: (value: string) => void;
 }
 
 const QRReader = ({ setQrData }: IQRReader) => {
   const [scanning, setScanning] = useState(false);
-  const [camera, setCamera] = useState("environment");
-
   const toogleScaning = () => setScanning(!scanning);
-  const toggleCamara = () => setCamera(camera === "environment" ? "user" : "environment")
+  const cameraMode = "environment"
```
Comment (Chinaski1979) [inline src/components/QRReader.tsx:15]: why is this equal to a static string?

File: src/context/AuthContext.tsx [hunk 4]
```diff
@@ -156,28 +99,176 @@ const AuthProvider = ({ children }: IProps) => {
...
-  )
-}
+  );
+};
+
+export default AuthProvider;
+
+
+
+// import React, { createContext, useContext, useEffect, useReducer, useState } from 'react'
```
Comment (Chinaski1979) [inline src/context/AuthContext.tsx:124]: why is all this code commented out? Please let avoid merging chunks of commented out code

File: src/context/reducers/authReducer.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import { developmentLogger } from '../../utils/developmentLogger';
```
Comment (Chinaski1979) [inline src/context/reducers/authReducer.ts:1]: Fix this file's indentation

File: src/components/QRReader.tsx (hunk 3 above)
Comment (jeezOC) [reply src/components/QRReader.tsx:15]: this was to toggle between front and back facing cameras, but this is more likely to be used just with the back camera, also the button was removed before the function was doing anything at all

File: src/context/AuthContext.tsx (hunk 4 above)
Comment (jeezOC) [reply src/context/AuthContext.tsx]: already removed 👍🏼

**Pull request merged**